### ISSUE TITLE
Is it possible to disable respawn fixer  ?

### ISSUE THREAD

**Issue by AIGLE25** [tomkeuper/BedWars1058#224]
**Is your feature request related to a problem? Please describe.**

Sometime when i die , i get tp too fast so i can't know what killed me and don't know if a player was hiding near to me  ,it makes you feel like you're dying for nothing  , some players reported that to me.

**Describe the solution you'd like**

Add a option to disable respawn fixer

**Comment by Squesly** [comment]
Respawn Fixer is buggy yea sometimes you get died from nothing, In that time I was running on top bridge with 1HP and no one hit me fireball me or do anything its just chnage ur state to respawn. That's why I don't use Zuyte respawn fixer and use function Instant-Respawn in Custom Spigot.

**Comment by tomkeuper** [comment]
Respawn fixer? im not sure what you guys are talking about. Is there a video or some sort of evidence to work with?

**Comment by tomkeuper** [comment]
And just FYI. Players can only die if some sort of damage event is triggered by spigot

**Comment by Squesly** [comment]
https://github.com/tomkeuper/BedWars2023/pull/144
Here this guy pull-requested and u merged this code.

**Comment by tomkeuper** [comment]
Thats a different issue tho. Thats about the respawn screen showing when a player dies

**Comment by Squesly** [comment]
It's the same issue tho he got killed by no reason as he said cuz he can't see what killed him. Because the respawn fixer causing it and I'll show you the video today. So you will understand it

**Comment by tomkeuper** [comment]
It did announce it in chat right?

**Comment by AIGLE25** [comment]
> It did announce it in chat right?

not when a mob kill you or something else

**Comment by tomkeuper** [comment]
Can you provide a quick video for me to understand?

**Comment by AIGLE25** [comment]
https://github.com/tomkeuper/BedWars2023/assets/68445450/cfe7c9a7-51e4-4858-97d1-4f54badc4977

**Comment by tomkeuper** [comment]
But it says that you died... I cant add all death messages into the plugin. If you make an addon it should handle that

**Comment by AIGLE25** [comment]
Yeah , but died form what , It would be cool if i was not instantly tp when i die but it's not a big problem too , just a little bit annoying

**Comment by AIGLE25** [comment]
I noticed it may be due to servers lag , when i was playing , i had like 60 - 80 ping

**Comment by tomkeuper** [comment]
It might be possible to add a configurable delay before teleporting people to the middle. Or a way to disable it completely. So you die but you wont get tp'd to the middle

**Comment by AIGLE25** [comment]
> It might be possible to add a configurable delay before teleporting people to the middle. Or a way to disable it completely. So you die but you wont get tp'd to the middle

sound good !

**Comment by AIGLE25** [comment]
> It might be possible to add a configurable delay before teleporting people to the middle. Or a way to disable it completely. So you die but you wont get tp'd to the middle

any new ?

**Comment by AIGLE25** [comment]
> It might be possible to add a configurable delay before teleporting people to the middle. Or a way to disable it completely. So you die but you wont get tp'd to the middle

![image](https://github.com/tomkeuper/BedWars2023/assets/68445450/f734c5e3-da7d-4a1e-b4ef-8118ad679fdb)
i made a little code change in DamageDeathMove.java that disable tp , and seem working (not very optimised but work)
